.github/policies/resourceManagement.yml

```
@@ -64,46 +64,53 @@ configuration:
       - isOpen
       - hasLabel:
           label: stale
+      - hasLabel:
+          label: triaged
       - not:
           hasLabel:
             label: bot-exclude
       then:
       - removeLabel:
           label: stale
-      - addLabel:
-          label: needs-triage
+      - removeLabel:
+          label: triaged
       description: Unstale Open Issues that get comments
     - if:
       - payloadType: Issue_Comment
       - not: isOpen
       - hasLabel:
           label: stale
+      - hasLabel:
+          label: triaged
       - not:
           hasLabel:
             label: bot-exclude
       then:
-      - addLabel:
-          label: needs-triage
-      description: Apply needs-triage to closed stale issues that get comments
+      - removeLabel:
+          label: triaged
+      description: Remove label triaged from closed stale issues that get comments
     - if:
       - payloadType: Issues
       - labelAdded:
           label: needs-more-info
-      - hasLabel:
-          label: needs-triage
+      - not:
+          hasLabel:
+            label: triaged

```

This might be more brief:

```suggestion
      - isNotLabeledWith:
          label: triaged
```

---

No, it could not :) What you suggest is a syntax for another entity, and it does not work here. 
I also thought it would and my first version indeed used "isNotLabeledWith". I needed to change it to the current version because I got a parsing error.

---

:-O

Interresting that they decided to support shorthand syntaxt on some entity, but not on other...